instrumentation/servlet/request-2.3/request-2.3.gradle

```diff
@@ -24,6 +24,7 @@ testSets {
 
 dependencies {
   compileOnly group: 'javax.servlet', name: 'servlet-api', version: '2.2'
+  compile(project(':instrumentation:servlet:servlet-common'))
```

Comment:
@trask can you advice me? I currently get `java.lang.ClassNotFoundException: io.opentelemetry.auto.instrumentation.servlet.ServletHttpServerTracer` during smoke tests. How instrumentations can share classes in general? If they can...

Reply:
I got it :)